File: 1347. Minimum Number of Steps to Make Two Strings Anagram/minSteps.cpp
class Solution
{
public:
    int minSteps(string s, string t)
    {

        map<char, int> m;
        for (auto it : s)
        {
            m[it]++;
        }
        int cnt = 0;
        for (auto it : t)
        {
            if (m.find(it) != m.end())
            {
                m[it]--;
                if (m[it] < 0)
                    cnt++;
            }
            else
            {
                cnt++;
            }
        }
        return cnt;
    }
};

File: 1347. Minimum Number of Steps to Make Two Strings Anagram/minSteps.java
class Solution {
    public int minSteps(String s, String t) {

        int freq[] = new int[26];
        for (char c : s.toCharArray()) {
            freq[c - 'a']++;
        }
        for (char c : t.toCharArray()) {
            freq[c - 'a']--;
        }
        int sum = 0;
        for (int val : freq) {
            if (val < 0)
                sum += val;
        }

        return Math.abs(sum);

    }
}